### ISSUE TITLE
How do I include external `00--prelim.Rmd` file in the `yaml` header?

### ISSUE THREAD

**Issue by atanasj** [ismayc/thesisdown#166]
The `00--prelim.Rmd` file has a number of headers. How is this called in the `yaml` header?

I have tried to put it under a heading e.g., `acknowledgements`, but then all the subheadings included in the document are printed in the final output i.e., `Acknowledgements>Preface`, `Acknowledgements>Acknowledgements`, etc. They way I interpreted the commented instructions I thought something along the lines of:

```yaml
acknowledgements: |
  '`r if(knitr:::is_latex_output()) paste(readLines(here::here("prelims", "00--prelim.Rmd")), collapse = "\n  ")`'
```

Although the all the content is printing, it is all under the `acknowledgements` heading.

Thanks in advance for your help.

**Comment by atanasj** [comment]
Okay, great. I got  that working by adding latex commands for the `toc` etc to specific docs, so I could make them appear where I wanted. 

Apologies that I am turning this into a help thread, however where would I set the font size of the headers and paragraph indentation. I need to format this as APA v7, and they have specific formatting requirements for header levels and paragraph indentation.

Thanks in advance.
